.github/workflows/push-trigger.yml

```diff
@@ -44,15 +44,15 @@ jobs:
     secrets:
       OSSRH_USER: ${{ secrets.OSSRH_USER }}
       OSSRH_SECRET: ${{ secrets.OSSRH_SECRET }}
-      OSSRH_URL: ${{ secrets.OSSRH_SNAPSHOT_URL }}
+      OSSRH_URL: ${{ secrets.RELEASE_URL }}
```

Comment:
it should be OSSRH_URL: ${{ secrets.OSSRH_SNAPSHOT_URL }} only

Reply:
made the change here --> https://github.com/mosip/digital-credential-plugins/pull/61